tests/python/pants_test/base/test_exclude_target_regexp_integration.py

```diff
@@ -1,146 +0,0 @@
-# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
I'm not sure it's safe to delete this test. The global option still exists. We could probably make it much simpler, though. Set up a `list` command and run it without the option, then with the option. Very similar to this:

https://github.com/pantsbuild/pants/blob/396ec642d612a91f582fc7ac996919cddc0ff83d/tests/python/pants_test/integration/build_ignore_integration_test.py#L14-L26

Reply:
Yea. The option was covered elsewhere, including in `tests/python/pants_test/integration/changed_integration_test.py`. This test was 100% JVM specific for some reason.